.github/workflows/ccruntime_e2e.yaml

```diff
@@ -55,6 +55,13 @@ jobs:
         env:
           TARGET_BRANCH: ${{ inputs.target-branch }}
 
+      - name: Login to Confidential Containers ghcr.io
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@stevenhorsman I think here we will need to create the secret

Reply:
The GITHUB_TOKEN is automatically created, so we don't need to make it.